packages/mobile/src/home/WalletHome.tsx

```diff
@@ -104,6 +113,13 @@ export class WalletHome extends React.Component<Props, State> {
     }
 
     ValoraAnalytics.setUserAddress(this.props.address)
+    const userInfo = {
+      accountAddress: this.props.accountAddress,
+      walletAddress: this.props.address,
+      cur: this.props.currency,
+      name: this.props.name,
+    }
+    ValoraAnalytics.addUserProfile(this.props.accountAddress, userInfo)
```

Comment:
Note for later, this logic shouldn't be tied to `WalletHome` and moved to a saga.

Reply:
moved!